### PR TITLE
New package: signond-8.58

### PR DESCRIPTION
**Pull request by subsonik**
Build requirement for kaccounts-integration (KDE Plasma 5).

### CODE REVIEW

File: srcpkgs/signond/template [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Template file for 'signond'
+
+pkgname="signond"
+version="8.58"
+revision=1
+build_style=qmake
+short_desc="SignOn daemon"
```
Comment (Vaelatern) [inline srcpkgs/signond/template]: A better description would be nice.

Comment (subsonik) [inline srcpkgs/signond/template]: Description updated.

File: srcpkgs/signond/template [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+signond-devel_package() {
+	short_desc+=" - development files"
+	depends="${sourcepkg}>=${version}_${revision}"
+	pkg_install() {
+		vmove usr/include
+		vmove usr/lib/*.so
+		vmove usr/lib/pkgconfig
+		vmove usr/share/doc
+		vmove usr/lib/*.a
+	}
```
Comment (xtraeme) [inline srcpkgs/signond/template]: You are missing `vmove usr/lib/cmake`

File: srcpkgs/signond/template [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+wrksrc="${pkgname}-VERSION_${version}-aa1bcf3c9218addbdb376a40151b689409046125"
+configure_args="LIBDIR=/usr/lib"
+
+post_install() {
+	vlicense COPYING
+}
+
+signond-devel_package() {
+	short_desc+=" - development files"
+	depends="${sourcepkg}>=${version}_${revision}"
```
Comment (xtraeme) [inline srcpkgs/signond/template]: `depends="${makedepends} ${sourcepkg}>=${version}_${revision}"`

File: srcpkgs/signond/template [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+homepage="https://gitlab.com/accounts-sso/signond"
+distfiles="https://gitlab.com/accounts-sso/${pkgname}/repository/archive.tar.gz?ref=VERSION_${version}>{pkgname}-{version}.tar.gz"
+checksum="301e2e3b6919585c8ffc56f73ec1f0f9b18a533e1db88461e9fa4e2acf75a34a"
+hostmakedepends="pkg-config doxygen"
+makedepends="qt5-devel"
+wrksrc="${pkgname}-VERSION_${version}-aa1bcf3c9218addbdb376a40151b689409046125"
+configure_args="LIBDIR=/usr/lib"
+
+post_install() {
+	vlicense COPYING
```
Comment (xtraeme) [inline srcpkgs/signond/template]: GPL licenses do not need to installed on disk, you can drop `post_install` completely.

File: srcpkgs/signond/template [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+pkgname="signond"
+version="8.58"
+revision=1
+build_style=qmake
+short_desc="A daemon providing the SSO service over D-Bus for the gSSO framework"
+maintainer="John Rowley <enterthevoid@codesector.co>"
+license="LGPL-2.1"
+homepage="https://gitlab.com/accounts-sso/signond"
+distfiles="https://gitlab.com/accounts-sso/${pkgname}/repository/archive.tar.gz?ref=VERSION_${version}>{pkgname}-{version}.tar.gz"
```
Comment (pullmoll) [inline srcpkgs/signond/template]: The rename should read: `>${pkgname}-${version}.tar.gz`

**Comment by subsonik** [comment]
Updated, thanks for your feedback.

File: srcpkgs/signond/template [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+maintainer="John Rowley <enterthevoid@codesector.co>"
+license="LGPL-2.1"
+homepage="https://gitlab.com/accounts-sso/signond"
+distfiles="https://gitlab.com/accounts-sso/${pkgname}/repository/archive.tar.gz?ref=VERSION_${version}>${pkgname}-${version}.tar.gz"
+checksum="301e2e3b6919585c8ffc56f73ec1f0f9b18a533e1db88461e9fa4e2acf75a34a"
+hostmakedepends="pkg-config doxygen"
+makedepends="qt5-devel"
+wrksrc="${pkgname}-VERSION_${version}-aa1bcf3c9218addbdb376a40151b689409046125"
+configure_args="LIBDIR=/usr/lib"
+
```
Comment (xtraeme) [inline srcpkgs/signond/template]: This is also missing `conf_files="/etc/signond.conf"`

**Comment by Vaelatern** [comment]
Thank you for your contribution! Please rebase this commit on top of master.

**Comment by subsonik** [comment]
Well that rebase went a bit wrong.. oops!

**Comment by Vaelatern** [comment]
Can you make a separate `-doc` subpackage? 1069 files in `usr/share/doc`.

**Comment by subsonik** [comment]
Done. TIL: separate -doc packages are a thing (never noticed!).

I've gotta say, I'm impressed by the level of attention you guys are showing to packaging. Split packages is one of the reasons I prefer Void over Arch. Awesome.

**Comment by Vaelatern** [comment]
Doesn't yet run as packaged.

**Comment by Vaelatern** [comment]
Okay, this looks good to me, but I think it needs to be: 1 commit, at revision one as it currently is, and rebased on master to clear any conflicts.

**Pull request closed without merging**